whydJS/app/controllers/LibUser.js

```diff
@@ -469,8 +469,9 @@ function renderUserLinks(lnk){
 
 	// make sure URLs are valid
 	for (var i in lnk)
-		if (lnk[i])
+		if (lnk[i] && i !== "home") {
```

Comment:
Nice fix, but it opens the door to `javascript:` URLs => vulnerability...

Need to make sure that the user-provided URL is either http:// or https:// protocol.